apps/partner_profile/routes.coffee

```diff
@@ -60,6 +64,27 @@ partnerFromProfile = (req) ->
     sectionLabel: "Articles"
     profile: req.profile
 
+@article = (req, res, next) ->
+  article = new Article id: req.params.articleId
+  article.fetch
+    cache: true
+    error: -> next()
```

Comment:
We have a helper `error: res.backboneError` that does a better job of parsing Backbone model error objects and determining how to respond. That said, there are good reasons to simply pass control to the next middleware like this. For instance if you're trying to gracefully catch and silence a 401 unauthed error, than you might want to do

```coffeescript
error: (err) ->
  if (err.status is 401) then next() else res.backboneError(err, next)
```

I'm not sure of the use case just looking at the code, but since the worst that would happen is rendering a 404—instead of a 500—page, I won't consider it blocking—just a heads up.